### ISSUE TITLE
Change text on report this request form to reflect type of material being reported

### ISSUE THREAD

**Issue by RichardTaylor** [mysociety/alaveteli#6390]
There is now a report button on each message and annotation. 

The text of the report page states, in all cases, irrespective of if what is being reported is a request, follow-up, response or annotation:

>Why specifically do you consider this request unsuitable?

The heading for the page does change depending on the type of material being reported. 

The word "request" could be switched for "material" for a simpler fix than actually specifying the type of material being reported.

**Comment by garethrees** [comment]
> The word "request" could be switched for "material" for a simpler fix than actually specifying the type of material being reported.

Yeah, this string is only used [in one place](https://github.com/mysociety/alaveteli/blob/c1fe79aa1168cd6f28d55ad52fe19bdb4b43f3e3/app/views/reports/new.html.erb#L12).

Really we should have a `Reportable` concern which allows us to dynamically get the type of object that's being reported…

```erb
<!-- something like this… -->
<p>
  <%= _('Why specifically do you consider this {{reportable}} unsuitable?',
        reportable: @reportable.class.humanize) %>
</p>
```

…but as it is we'd have to "hack" it with something like `(@comment || @outgoing_message || @incoming_message || @info_request).class.humanize` – not terrible, but more complicated and I think the text change works just as well here.
